test/base/SpecBase.scala

```diff
@@ -70,11 +70,17 @@ trait SpecBase
 
   val emptyUserAnswers: UserAnswers = UserAnswers(userAnswersId, draftId)
 
-  val userAnswersAsIndividualTrader: UserAnswers = emptyUserAnswers
-    .setFuture(AccountHomePage, IndividualTrader)
-    .futureValue
+  val userAnswersAsIndividualTrader: UserAnswers =
+    emptyUserAnswers
+      .setFuture(AccountHomePage, IndividualTrader)
+      .futureValue
+
+  val userAnswersAsOrgAdmin: UserAnswers =
+    emptyUserAnswers
+      .setFuture(AccountHomePage, OrganisationAdmin)
+      .futureValue
 
-  val userAnswersAsOrgAdmin: UserAnswers = emptyUserAnswers
+  emptyUserAnswers
     .setFuture(AccountHomePage, OrganisationAdmin)
     .futureValue
```

Comment:
This looks like duplication of userAnswersAsOrgAdmin. Maybe set emptyUserAnswers to private to force the other user answers to be used?

Reply:
oops, yep a dupe